src/expandableCalendar/index.js

```diff
@@ -445,13 +445,7 @@ class ExpandableCalendar extends Component {
     return (
       <Animated.View
         ref={e => (this.weekCalendar = e)}
-        style={{
-          position: 'absolute',
-          left: 0,
-          right: 0,
-          top: HEADER_HEIGHT + (commons.isAndroid ? 8 : 4), // align row on top of calendar's first row
-          opacity: position === POSITIONS.OPEN ? 0 : 1
-        }}
+        style={[this.style.weekContainer, {opacity: position === POSITIONS.OPEN ? 0 : 1}]}
```

Comment:
Also here, you can create styles for `hidden` (0) and visible (1) and use it in the ternary. 
even better, the default opacity is 1, so instead you can just do `position === POSITIONS.OPEN && styles.hidden`, no need for ternary

Reply:
Done